test/unit/oracle-aggregator.spec.ts

```diff
@@ -220,52 +210,6 @@ describe('OracleAggregator', () => {
     });
   });
 
-  describe('addSupportForPairIfNeeded', () => {
```

Comment:
I'm not entirely sure why are we getting rid of all these tests?  We are testing custom behavior of `OracleAggregator.addSupportForPairIfNeeded`

Reply:
We are deleting the test, because we haven't coded that function. That function is now implemented in Simple Oracle, and it has already been tested there

Reply:
I mean... We are overriding **when**  its called. Although in  #40 you will test more thoroughly this, so I am ok with this.

Reply:
I don't get the distinction. We deleted the test because we also deleted the function. It's now implemented in SimpleOracle, and we also test (in a test for SimpleOracle) that it calls the internal function when expected.

We do test the internal function here, so I believe everything is covered. Maybe I'm missing something?